src/Compilers/CSharp/Portable/Syntax/SyntaxFactory.cs

```diff
@@ -2456,7 +2456,7 @@ public static BlockSyntax Block(IEnumerable<StatementSyntax> statements)
             SyntaxToken operatorKeyword,
             TypeSyntax type,
             ParameterListSyntax parameterList,
-            BlockSyntax body,
+            BlockSyntax? body,
```

Comment:
I assume this one is because we can have conversion operators in interfaces now, without a body?

Reply:
Also Metadata As Source does not display body.